devel/sift/Portfile

```diff
@@ -0,0 +1,80 @@
+				adding features while trying to reach (or even surpass) the performance \
+				of the original grep. The additional features include gitignore support, \
+				conditions (e.g., match A only when preceded by B within X lines), \
+				full multi-core support and multiline matching.
+homepage        http://sift-tool.org
+platforms       darwin
+use_zip          yes
+
+master_sites	https://github.com/svent/sift/archive:sift \
+				https://github.com/jessevdk/go-flags/archive:flags \
+				https://github.com/svent/go-nbreader/archive:nbreader \
+				https://raw.githubusercontent.com/golang/crypto/master/ssh/terminal:crypto
+distfiles		v${version}.zip:sift \
+				terminal.go:crypto \
+				util_bsd.go:crypto \
+				util.go:crypto \
+				v1.1.0.zip:flags \
+				master.zip:nbreader
+				
+checksums       v${version}.zip \
+					rmd160 e1b1eae2e314510b852ee86f0152fa541b5b4af9 \
+	                sha256  00f4a57cd4140999443833b6bbf446a21ec5660613af5ce101d651a265152051 \
+	            util.go \
+	            	rmd160 6f17dbf32422a7e43d9182137c0ebc2bc9704ef5 \
+	            	sha256 c0506e16f0f7ec105437f35f2a7fb6c3b71a28f95102f3b5dec0d22f040cb727 \
+	            util_bsd.go \
+	            	rmd160 9a75a1dc87ad32232627302e266215cba35766a4 \
+	            	sha256 fc3b7bafbded5b824b3b9c5d4c4d6730958cbeb9ae9cd2dde2132d670d2cc3f4 \
+	            terminal.go \
+            		rmd160 ad3f5da181a2fa618e104b1deb79a869a9164cb2 \
+	            	sha256 b35ec763e014dcc944989ff5ddea401c754bcc27c011f71012a2454eaca1b11a \
+	            v1.1.0.zip \
+	            	rmd160 19254b903fe3313354ad01ddc8b3923f78ef2476 \
+	            	sha256 1bd4c25d478fe661761be24d51ae842bcea1369f79c8b485ecbb46ce17a9a46e \
+	            master.zip \
+					rmd160 555fbce2dd0db3ecc8ac4d25baafcc7f68e5f427 \
+	            	sha256 cad400afa41d6f5e08d70808270425a016f5a54c30bafd554dcdc2b6161e0288
+
+extract.only	v${version}.zip v1.1.0.zip master.zip
+
+depends_build	    port:go
+universal_variant   no
+use_configure       no
+
+build.cmd           go
+build.target        build
+build.env           GOPATH=${worksrcpath}
+
+post-extract {
+	file mkdir ${worksrcpath}/src/golang.org/x/crypto/ssh/terminal
+	ln -sf ${distpath}/terminal.go ${worksrcpath}/src/golang.org/x/crypto/ssh/terminal/terminal.go
+	ln -sf ${distpath}/util_bsd.go ${worksrcpath}/src/golang.org/x/crypto/ssh/terminal/util_bsd.go
+	ln -sf ${distpath}/util.go ${worksrcpath}/src/golang.org/x/crypto/ssh/terminal/util.go
+	file mkdir ${worksrcpath}/src/github.com/svent
+	file mkdir ${worksrcpath}/src/github.com/svent/sift
+	move ${worksrcpath}/gitignore ${worksrcpath}/src/github.com/svent/sift
+	move ${workpath}/go-nbreader-master ${worksrcpath}/src/github.com/svent/go-nbreader
+	move ${workpath}/go-flags-1.1.0 ${worksrcpath}/src/github.com/svent/go-flags
+}
+
+destroot {
+	move ${worksrcpath}/${name}-${version} ${worksrcpath}/${name}
+	xinstall ${worksrcpath}/${name} ${destroot}${prefix}/bin
```

Comment:
I would recommend at least `xinstall -m755` to ensure this is installed with the executable bits set.